python/tests/expressions/test_expressions_base.py

```diff
@@ -139,210 +217,138 @@ def test_strs(op, string):
     assert str(op) == string
 
 
+def test_ref_binding_case_sensitive(request):
+    schema = request.getfixturevalue("table_schema_simple")
+    ref = base.Reference("foo")
+    bound = base.BoundReference(schema.find_field(1), schema.accessor_for_field(1))
+    assert ref.bind(schema, case_sensitive=True) == bound
+
+
+def test_ref_binding_case_sensitive_failure(request):
+    schema = request.getfixturevalue("table_schema_simple")
+    ref = base.Reference("Foo")
+    with pytest.raises(ValueError):
+        ref.bind(schema, case_sensitive=True)
+
+
+def test_ref_binding_case_insensitive(request):
+    schema = request.getfixturevalue("table_schema_simple")
+    ref = base.Reference("Foo")
+    bound = base.BoundReference(schema.find_field(1), schema.accessor_for_field(1))
+    assert ref.bind(schema, case_sensitive=False) == bound
+
+
+def test_ref_binding_case_insensitive_failure(request):
+    schema = request.getfixturevalue("table_schema_simple")
+    ref = base.Reference("Foot")
+    with pytest.raises(ValueError):
+        ref.bind(schema, case_sensitive=False)
+
+
+def test_in_to_eq():
+    assert base.In(base.Reference("x"), (literal(34.56),)) == base.EqualTo(base.Reference("x"), literal(34.56))
+
+
+def test_bind_in(request):
+    schema = request.getfixturevalue("table_schema_simple")
+    bound = base.BoundIn(
+        base.BoundReference(schema.find_field(1), schema.accessor_for_field(1)), {literal("hello"), literal("world")}
+    )
+    assert base.In(base.Reference("foo"), (literal("hello"), literal("world"))).bind(schema) == bound
+
+
+def test_bind_dedup(request):
+    schema = request.getfixturevalue("table_schema_simple")
```

Comment:
It's definitely not required but instead of using `request` here you can just use the fixture name and pytest will automatically grab it for you.
```py
def test_bind_dedup(table_schema_simple):
    bound = base.BoundIn(
        base.BoundReference(table_schema_simple.find_field(1), table_schema_simple.accessor_for_field(1)), {literal("hello"), literal("world")}
    )
    assert base.In(base.Reference("foo"), (literal("hello"), literal("world"), literal("world"))).bind(table_schema_simple) == bound
```